app/src/main/java/com/hedvig/app/feature/marketing/data/GetInitialMarketPickerValuesUseCase.kt

```diff
@@ -39,6 +39,6 @@ class GetInitialMarketPickerValuesUseCase(
       .fold({ null }, ::identity)
       ?: Market.SE
 
-    return Pair(marketOrDefault, null)
+    return Pair(marketOrDefault, marketOrDefault.defaultLanguage())
```

Comment:
I wonder if we relied on this null behavior somewhere in a way that was meaningful. If not I think this leaves this function never returning null for language, so the signature can change to be a non-null Language then